dtreeviz/trees.py

```diff
@@ -176,8 +177,12 @@ def ctree_leaf_distributions(self,
             Plot labels font name
         :param grid: bool
             True if we want to display the grid lines on the visualization
-        :param figsize: optional (width, height) in inches for the entire plot
-            :param ax: optional matplotlib "axes" to draw into
+        :param figsize: optional
+            (width, height) in inches for the entire plot
+        :param ax: optional
+            matplotlib "axes" to draw into
+        :param label_all_leaves: bool
```

Comment:
I think I have standardized figsize, ax to be the final two parameters so perhaps move yours up?